src/control/cmd/dmg/pool_test.go

```diff
@@ -88,6 +88,75 @@ func TestPoolCommands(t *testing.T) {
 	}
 
 	runCmdTests(t, []cmdTest{
+		{
+			"Create pool with label prop and flag",
```

Comment:
How about the case with the flag and the positional argument? Right now it looks like the flag overrides the positional arg

Reply:
I went back and forth on this... It seemed excessive to add more code to check for this corner case, particularly given that I am hoping to remove the --label flag in a future release. In the end I settled on the (perhaps self-serving) conclusion that if someone specifies labels by both the flag and the argument, the flag takes precedence.